docs/examples/cloudpubsubsource/README.md

```diff
@@ -1,4 +1,104 @@
 # CloudPubSubSource Example
 
-Please refer to the [example](https://knative.dev/docs/eventing/samples/gcp-pubsub-source/) in Knative until we migrate
-it to this repo. 
+## Overview
+
+This sample shows how to configure the CloudPubSubSource. 
+This source is most useful as a bridge from other GCP services, 
+such as [Cloud Storage](https://cloud.google.com/storage/docs/pubsub-notifications), 
+[Cloud Scheduler](https://cloud.google.com/scheduler/docs/creating#).
```

Comment:
I would remove this, as it might be confusing... we already have sources for those...